src/UI/state/home/actions/getHomepageStats.ts

```diff
@@ -0,0 +1,23 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { fetchGraphQlData } from '../../../api/api';
+import { getHomepageAnalytics } from '../../../api/queries';
+import { updateStats, serverValidation } from '../homeSlice';
+
+const date = new Date();
+
+export const getHomepageStats = createAsyncThunk(
+  'home/getHomepageStats',
+  async (_, thunkAPI) => {
+    try {
+      const response = await fetchGraphQlData(
+        getHomepageAnalytics(0, date.getTime(), 'hour', 'Europe%2FLondon')
+      );
+      const data = response.data.getHomepageAnalytics;
+      thunkAPI.dispatch(serverValidation(true));
```

Comment:
Is this similar to a `statsLoading` flag? If so, can we change it to be a loading flag instead? It'd be more consistent with the rest of the actions.